CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ that can be found in the LICENSE file. -->
 
 # CHANGELOG
 
+## 2.8.0
+
+### Feature
+
+- Support android API 34(Android 14) limit access to photos and videos.
+- Because limit permission, we refactor the permission request API.
+
+***Breaking changes for permission behavior***
```

Comment:
Is it an actual behavior change? I don't see the difference base on the description.

Reply:
Good to know. I don't think the phrase tells the fact properly. Consider this:

```markdown
Methods do not implicitly call for permission requests anymore.
User must follow the below methods to ensure permissions were granted:

1. `PhotoManager.requestPermissionExtend()`, verify if the result is
   `PermissionState.authorized` or `PermissionState.limited`.
2. `PhotoManager.setIgnorePermissionCheck(true)`, ignoring permission checks,
   handle permission with other mechanisms.
```

Reply:
Ok, I will change the CHANGLOG.md.